interceptor/pkg/interceptor/pdinterceptor_test.go

```
@@ -223,105 +223,13 @@ func TestOversizedRequestBodyIsRejected(t *testing.T) {
 	req := httptest.NewRequest(http.MethodPost, "/", bytes.NewReader(oversizedBody))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)